src/gui/optionsdlg.ui

```diff
@@ -90,7 +90,7 @@
      </widget>
      <widget class="QStackedWidget" name="tabOption">
       <property name="currentIndex">
-       <number>0</number>
+       <number>5</number>
```

Comment:
PR looks good to me. nitpick though, it would be nice to edit this one back to 0.